app/services/cfe/submit_assets_service.rb

```diff
@@ -16,7 +16,7 @@ def call(cfe_assessment_id)
       if asset_form.property_value.positive?
         second_property = {
           value: asset_form.property_value,
-          outstanding_mortgage: asset_form.property_mortgage,
+          outstanding_mortgage: asset_form.property_mortgage.to_s.strip.to_f,
```

Comment:
might be worth testing to see if `to_f `will work on  its own. According to the docs it should.
https://apidock.com/ruby/v2_5_5/String/to_f